tests/lib/rule-context.js

```diff
@@ -95,4 +96,26 @@ describe("RuleContext", function() {
         });
     });
 
+    describe("parserServices", function() {
+
+        it("should pass through parserServices properties to context", function() {
+            const services = {
+                test: {}
+            };
+            const ruleContext = new RuleContext("fake-rule", {}, 2, {}, {}, {}, "espree", {}, services);
+
+            assert.equal(ruleContext.parserServices.test, services.test);
+        });
+
+        it("should copy parserServices properties to a new object", function() {
+            const services = {
+                test: {}
+            };
+            const ruleContext = new RuleContext("fake-rule", {}, 2, {}, {}, {}, "espree", {}, services);
+
+            assert.notEqual(ruleContext.parserServices, services);
+        });
```

Comment:
Any point in adding a test for freezing? (I dunno how you would, honestly, just wanted to raise the question.)